js/data.js

```diff
@@ -949,7 +949,7 @@ var data = {
     },
     personalManagementSkills: {
         kurz: "Personal<br />Management",
-        kennung: [SSKMA],
+        kennung: [SSKMA, SSKKO],
```

Comment:
Has not always been like this, for example in WS1415. Not sure, how thats handled, if someone, who took the course last year, can still put it under SSKKO.
So far, I'd not add this here.